modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
+
+[source,bash]
+----
+curl http://localhost:8080/movies
 ----
-public interface MovieRepository extends Neo4jRepository<Movie, Long> {
-    Movie getMovieByTitle(String title);
 
-    Iterable<Movie> findMovieByTitleLike(String title);
-}
+Results: retrieve all movies in our database
+
+* `localhost:8080/movies <movieToUpdateOrCreate>` for createOrUpdateMovie() method
+
+[source,bash]
 ----
+curl -X "PUT" "http://localhost:8080/movies" \
+     -H 'Content-Type: application/json; charset=utf-8' \
+     -d $'{
+  "title": "Aeon Flux",
+  "description": "Reactive is the new cool"
+}'
+----
+
+Results: create new movie `Aeon Flux` in our database
 
-As with our `PersonRepository` above, these methods in our `MovieRepository` will allow us to search for a specific movie title or retrieve a list of possible matches.
+* `localhost:8080/movies/by-title` for byTitle() method
+
+[source,bash]
+----
+curl http://localhost:8080/movies/by-title\?title\=Aeon%20Flux
+----
+
+Results: retrieve information about the specific movie (in this query, `Aeon Flux`)
+
+* `localhost:8080/movies/\{id\}` for delete() method
+
+[source,bash]
+----
+curl -X DELETE http://localhost:8080/movies/847
+----
+
+Results: delete the movie using its id (in this case, the `Aeon Flux` movie)
 
 [#sdn-resources]
 == Resources
 
-For a more thorough walkthrough of the code, see the resources linked below.
-
 [cols="1,4"]
 |===
-| icon:code-fork[] Projects | https://projects.spring.io/spring-data-neo4j/[Spring Data Neo4j^]
-| icon:user[] Authors | The Neo4j, http://graphaware.com/neo4j-experts/[GraphAware^], and Pivotal teams.
-| icon:gift[] Package | link:http://maven.org[http://maven.org]
+| icon:code-fork[] Projects | https://spring.io/projects/spring-data-neo4j/[Spring Data Neo4j^]
 | icon:github[] Source | https://github.com/spring-projects/spring-data-neo4j
 | icon:medkit[] Issues | https://jira.spring.io/browse/DATAGRAPH[JIRA^]
-| icon:book[] Docs | http://docs.spring.io/spring-data/data-neo4j/docs/current/reference/html/[Reference^], http://docs.spring.io/spring-data/data-neo4j/docs/current/api/[JavaDoc^], http://docs.spring.io/spring-data/data-neo4j/docs/current/changelog.txt[ChangeLog^]
-| icon:book[] Articles | http://graphaware.com/blog/sdn/[GraphAware^], https://medium.com/neo4j/neo4j-ogm-and-spring-data-neo4j-a55a866df68c[SDN and OGM^]
-| icon:film[] Video | https://www.youtube.com/watch?v=u4YYuQ-Zook[Spring Data Neo4j 5 and OGM3^]
+| icon:book[] Docs | https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/[Reference^], http://docs.spring.io/spring-data/data-neo4j/docs/current/api/[JavaDoc^], https://docs.spring.io/spring-data/data-neo4j/docs/current/changelog.txt[ChangeLog^]
+| icon:book[] Articles | https://medium.com/neo4j/spring-data-neo4j-6-0-8b92164fff32[Introducing SDN 6^]
 | icon:play-circle[] Examples |https://github.com/spring-projects/spring-data-neo4j[SDN Example from Spring^], https://github.com/neo4j-examples?q=spring[Spring Data Neo4j^]
```

Comment:
We should cherry-pick the examples here. E.g. the migration example would not get matched by the query. Also we get in old SDN 5 examples.